joomla/joomla-cms
# [4.1] Use to PHP 8.1 fixes for debug bar dependency

Pull Request for Issue #37274.

### Summary of Changes
Updates the PHP debugbar dependency to the commit with the latest PHP 8.1 fixes. It is basically the release 1.18.0 with the fixes.

### Testing Instructions
- Run Joomla on PHP 8.1
- Enable debugging in the global Joomla configuration
- Set error reporting to maximum in the global Joomla configuration
- Make sure the debug plugin is enabled by default (is the case when doing a new installation)

### Actual result BEFORE applying this Pull Request
The following error is shown:
_Implicit conversion from float -INF to int loses precision in_

### Expected result AFTER applying this Pull Request
No error.
